holmes/plugins/toolsets/elasticsearch/elasticsearch.py

```
@@ -0,0 +1,632 @@
+                    type="string",
+                    required=False,
+                ),
+                "health": ToolParameter(
+                    description="Filter by index health (green, yellow, red). Only for indices endpoint.",
+                    type="string",
+                    required=False,
+                ),
+            },
+        )
+
+    def _invoke(self, params: dict, context: ToolInvokeContext) -> StructuredToolResult:
```

Ruff (0.14.10)

Unused method argument: `context`

(ARG002)